src/simulation.py

```diff
@@ -8,14 +8,15 @@
 class Simulation():
     def __init__(self):
         self.config = Config()
+        self.generation_duration = self.config.generation_duration
         self.world = World(self.config)
         self.population = Population(self.config)
         self.screen = screen_init(self.config)
         self.running = True
```

Comment:
I think this should be set as False in the constructor and only be true if self.run() is called.